main.js

```diff
@@ -0,0 +1,2 @@
+const nombreUsuario = prompt("Ingrese su nombre");
```

Comment:
En este caso el mensaje deberia ser "Ingresar el nombre".
```suggestion
const nombreUsuario = prompt("Ingresar el nombre");
```